src/gui/mainwindow.cpp

```diff
@@ -843,6 +843,7 @@ void MainWindow::createKeyboardShortcuts()
     m_ui->actionDelete->setShortcutContext(Qt::WidgetShortcut);  // nullify its effect: delete key event is handled by respective widgets, not here
     m_ui->actionDownloadFromURL->setShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_O);
     m_ui->actionExit->setShortcut(Qt::CTRL + Qt::Key_Q);
+    m_ui->actionClose_Window->setShortcut(QKeySequence::Close);
```

Comment:
I would make this macOS only.

Reply:
Reason is I certainly won't expect this key sequence "close qbt" (if I'm not mistaken) on windows/linux.
And also I didn't expect there to be another menu entry for this.